src/website/app/demos/Icon/index.js

```diff
@@ -16,14 +16,25 @@
 
 /* @flow */
 import { componentTheme } from '../../../../Icon/Icon';
+import bestPractices from './bestPractices';
 import examples from './examples';
 
 const doc = require('!!react-docgen-loader!../../../../Icon/Icon');
 
 export default {
+  bestPractices,
   componentTheme,
   doc,
   examples,
   slug: 'icon',
-  title: 'Icon'
+  title: 'Icon',
+  whenHowToUse: `Icons can symbolize actions and objects in your interace.
+Use icons in combination with labels to help users more quickly process your UI.
+
+Don't use too many icons or you run the risk of creating visual clutter.
+
+Icons are usually used inside of a [Button](./button) to reinforce a call to action.
```

Comment:
"...to reinforce the action."? CTAs are a more specific case.

Reply:
addressed